TechnologyAdvice/progress-bar-4-axios
# Update accessible nprogress

### Description
This PR is to test and eventually incorporate our new version of axios-progress-bar
### Related issue(s)
https://technologyadvice.atlassian.net/secure/RapidBoard.jspa?rapidView=16&projectKey=BT&view=detail&selectedIssue=BT-754&assignee=5e8c65719252d10b8a8efe7f
### Related PR(s)
https://github.com/TechnologyAdvice/out-task-portal/pull/139
### How to test / repro

- pull this branch and the related PR

- run npm install on the axios repo, and then run npm link (may need to use sudo npm link if you don't have permissions)

- in this out-task-portal branch, npm link axios-progress-bar (may also need sudo)

- npm run build in the axios branch, npm run dev in out task portal

- in the out-task-portal branch, run npm run test:e2e and click the accessibility tests

- They should complain only about needing labels on our tagoodtable inputs

### Screenshots

### Changes include

### Checklist
